armotypes/vulnerabilitytypes.go

```diff
@@ -82,23 +82,26 @@ type ComponentPathInfo struct {
 }
 
 type Vulnerability struct {
-	Name           string                     `json:"name"`
-	Severity       string                     `json:"severity"`
-	SeverityScore  int                        `json:"severityScore"`
-	IsRCE          bool                       `json:"isRCE"`
-	Links          []string                   `json:"links"`
-	Description    string                     `json:"description"`
-	Exploitable    string                     `json:"exploitable"`
-	ComponentsInfo []VulnerabilitiesComponent `json:"componentsInfo"`
-	IsFixable      bool                       `json:"isFixable"`
-	CvssInfo       CvssInfo                   `json:"cvssInfo"`
-	EpssInfo       EpssInfo                   `json:"epssInfo"`
-	CisaKevInfo    CisaKevInfo                `json:"cisaKevInfo"`
+	Name           string                   `json:"name"`
+	Severity       string                   `json:"severity"`
+	SeverityScore  int                      `json:"severityScore"`
+	Links          []string                 `json:"links"`
+	Description    string                   `json:"description"`
+	Exploitable    string                   `json:"exploitable"`
+	InUse          bool                     `json:"inUse"`
```

Comment:
let's keep it as relevant and FE will translate, because on the workload obj the feild name is hasrelevancyData